apps/web/components/dashboard/preview/AssetContentSection.tsx

```diff
@@ -1,42 +1,92 @@
+import { useEffect, useState } from "react";
 import Image from "next/image";
 import Link from "next/link";
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
 
 import { BookmarkTypes, ZBookmark } from "@hoarder/shared/types/bookmarks";
 
 export function AssetContentSection({ bookmark }: { bookmark: ZBookmark }) {
+  const [section, setSection] = useState<string>("");
+
+  useEffect(() => {
+    const screenshot = bookmark.assets.find(
+      (item) => item.assetType === "screenshot",
+    );
+    if (screenshot) {
+      setSection("screenshot");
+    } else {
+      setSection("pdf");
+    }
+  }, [bookmark]);
```

Comment:
As I mentioned above, this might not be worth it, but if we're going that route, `useMemo` is more correct here than `useEffect`